File: android/source/VideoLocker/src/org/edx/mobile/view/registration/RegistrationTextView.java
package org.edx.mobile.view.registration;

import android.text.InputType;
import android.view.View;

import org.edx.mobile.model.registration.RegistrationFormField;

/**
 * Created by rohan on 2/11/15.
 */
public class RegistrationTextView extends RegistrationEditTextView {

    public RegistrationTextView(RegistrationFormField field, View view) {
        super(field, view);
        mInputView.setInputType(InputType.TYPE_CLASS_TEXT);
    }
}

File: OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/view/RegistrationTextView.kt
package org.edx.mobile.module.registration.view

import android.text.InputType
import android.view.View
import org.edx.mobile.module.registration.model.RegistrationFormField

internal class RegistrationTextView(
    field: RegistrationFormField,
    view: View,
) : RegistrationEditTextView(field, view) {
    init {
        mBinding.editText.inputType = InputType.TYPE_CLASS_TEXT
    }
}
